rules/windows/defense_evasion_clearing_windows_event_logs.toml

```diff
@@ -11,33 +11,34 @@ attackers in an attempt to evade detection or destroy forensic evidence on a sys
 """
 from = "now-9m"
 index = ["winlogbeat-*", "logs-endpoint.events.*", "logs-windows.*"]
-language = "kuery"
+language = "eql"
 license = "Elastic License v2"
 name = "Clearing Windows Event Logs"
 risk_score = 21
 rule_id = "d331bbe2-6db4-4941-80a5-8270db72eb61"
 severity = "low"
 tags = ["Elastic", "Host", "Windows", "Threat Detection", "Defense Evasion"]
 timestamp_override = "event.ingested"
-type = "query"
+type = "eql"
 
 query = '''
-event.category:process and event.type:(process_started or start) and 
-(process.name:"wevtutil.exe" or process.pe.original_file_name:"wevtutil.exe") and 
-process.args:("/e:false" or cl or "clear-log") or process.name:"powershell.exe" and process.args:"Clear-EventLog"
+process where event.type in ("process_started", "start") and
+  (process.name : "wevtutil.exe" or process.pe.original_file_name == "wevtutil.exe") and
```

Comment:
@brokensound77 Hey, just curious. I guess I don't understand why on same line one field (`process.name`) gets the `:` and the other field `process.pe.original_file_name` gets `==`.  Is this due to some limitation with `original_file_name` field? This [rule ](https://github.com/elastic/detection-rules/blob/414d32027632a49fb239abb8fbbb55d3fa8dd861/rules/windows/discovery_security_software_wmic.toml) appears to use the `:` for `original_file_name`

Reply:
`==` is a stricter check since it is case sensitive. Original file name casing is parsed consistently according to the actual casing used when a binary was compiled, whereas the casing of the process or args are parsed based in how the user called them.

Reply:
Ok, cool.  Thanks for the clarification.